src/php/IFSC.php

```diff
@@ -56,19 +62,27 @@ public static function validate($code)
      */
     public static function validateBankCode($bankCode)
     {
-        self::init();
-        return array_key_exists($bankCode, self::$data);
+        return defined("Razorpay\IFSC\Bank::$bankCode");
```

Comment:
We can call init here, but it is not needed, since the `defined()` check doesn't read any of the variables defined in there, so it is optional here.